tests/functional/worker/all-uncategorized.test.js

```diff
@@ -193,7 +193,7 @@ function harvestSessionIsNullWhenEnabled (type, browserVersionMatcher) {
       Promise.all([ajaxPromise, loadPromise, router.expectRum()])
         .then(([{ request: { query } }]) => {
           t.equal(query.ck, '0', "The cookie flag ('ck') should equal 0.")
-    		t.equal(query.s, '0', "The session id attr 's' should be 0.")
+    		t.ok(query.s, "The session id attr 's' should exist and have a truthy value.")
```

Comment:
Why is worker sending session id now?

Reply:
Is there a reason it shouldn't? We didn't before because we were limited. The limitation is gone so I guess it's a question of "why not"?

Reply:
do not spawn session entity if cookies disabled in settings